deployment/ccip/view/v1_6/rmnremote.go

```diff
@@ -22,6 +31,35 @@ type RMNRemoteSigner struct {
 	NodeIndex        uint64 `json:"node_index"`
 }
 
+func mapCurseSubjects(subjects [][16]byte) []RMNRemoteCurseEntry {
+	res := make([]RMNRemoteCurseEntry, 0, len(subjects))
+	for _, subject := range subjects {
+		res = append(res, RMNRemoteCurseEntry{
+			Subject:  hex.EncodeToString(subject[:]),
+			Selector: SubjectToSelector(subject),
+		})
+	}
+	return res
+}
+
+var (
+	globalSubject = [16]byte{0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01}
+)
+
+func SubjectToSelector(subject [16]byte) uint64 {
+	if subject == globalSubject {
+		return 0
+	}
+
+	return binary.BigEndian.Uint64(subject[8:])
+}
+
+func SelectorToSubject(selector uint64) [16]byte {
+	subject := globalSubject
+	binary.BigEndian.PutUint64(subject[8:], selector)
+	return subject
+}
```

Comment:
Looks like these are duplicates from deployment/ccip/changeset/v1_6/cs_rmn_curse_uncurse.go? can you make a common helper ?